commands/invoke.go

```diff
@@ -21,8 +21,8 @@ var (
 
 func init() {
 	// Setup flags that are used by multiple commands (variables defined in faas.go)
-	invokeCmd.Flags().StringVar(&gateway, "gateway", defaultGateway, "Gateway URI")
 	invokeCmd.Flags().StringVar(&functionName, "name", "", "Name of the deployed function")
```

Comment:
@alexellis since we are removing the flag and handling it via argument passed to invoke. do we need to see this flag `--name`. also this is in new_function.go